.ebextensions/cronjob.config

```diff
@@ -28,15 +28,22 @@ files:
         owner: root
         group: root
         content: |
-            45 3 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake data_feeds:meteostat_loader >> log/jobs.log 2>&1'
+            50 3 * * * root /bin/bash -l -c 'cd /var/app/current && RAILS_ENV=production bin/run_as_webapp bundle exec rake data_feeds:meteostat_loader >> log/jobs.log 2>&1'
 
-    "/etc/cron.d/back-fill-any-dark-sky-temperatures":
+    "/etc/cron.d/back-fill-any-meteostat-temperatures":
```

Comment:
Are these jobs named the right way round (meteostat and dark sky backfills)..?

Reply:
d'oh, edited the wrong line